newton/tests/test_lookdev_usd.py

```
@@ -0,0 +1,340 @@
+        b = newton.ModelBuilder()
+        b.add_ground_plane()
+        b.add_body(
+            xform=wp.transform(wp.vec3(0.0, 0.0, 2.0), wp.quat_identity()),
+            mass=1.0,
+            inertia=wp.mat33(1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0),
+        )
+        b.add_shape_sphere(body=0, radius=0.5)
+        return b.finalize()
+
+    def _viewer(self, tmp_name, lookdev):
+        path = os.path.join(os.environ.get("CLAUDE_JOB_DIR", "/tmp"), "tmp", tmp_name)
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)